src/config.js

```diff
@@ -764,7 +847,13 @@ class Config {
   // Extended: Get an {Array} of all of the `source` {String}s with which
   // settings have been added via {::set}.
   getSources () {
-    return _.uniq(_.pluck(this.scopedSettingsStore.propertySets, 'source')).sort()
+    const arrs = []
+    arrs.push(_.uniq(_.pluck(this.globalSettings.scopedSettings.propertySets, 'source')))
```

Comment:
are these `uniq`s needed if there's one that takes place after a flattening below?